### PR TITLE
fix(styles): add styles for form inputs

### PR DESCRIPTION
**Pull request by imranhsayed**
#### Description of what you did:

Adds styles for inputs in the content to support forms

#### My PR is a:

- 🔝 Improvement

#### Is the PR ready to be merged?

- ✅ Yes!

### CODE REVIEW

**Comment by imranhsayed** [comment]
@luisherranz Can you review the PR please.

File: packages/mars-theme/src/components/post.js [hunk 1]
```diff
@@ -141,6 +141,49 @@ const Content = styled.div`
     text-decoration: underline;
   }
 
+  // Add input fields styles.
```
Comment (luisherranz) [inline packages/mars-theme/src/components/post.js]: This comment is not valid so please either remove it or use a CSS comment syntax: `/* Add input fields styles. */`

**Review by luisherranz — DISMISSED** [review]

**Comment by luisherranz** [comment]
@DAreRodz could you please review the CSS and if appropriate, approve and merge this PR before today's release? Thanks!

EDIT: You can remove the comment yourself and remember to add a proper title for squashing.

**Comment by DAreRodz** [comment]
Of course, no problem.  👍

**Review by DAreRodz — APPROVED** [review]
Good job, Imran!  👍

**Pull request merged**